File: scripts/generate_600_changes.py
from pathlib import Path
p = Path(__file__).resolve().parent.parent / 'mass_changes'
p.mkdir(parents=True, exist_ok=True)
for i in range(1, 601):
    name = f'change_{i:03}.txt'
    content = f'Change file {i:03}\nGenerated for commit #{i}\n'
    (p / name).write_text(content, encoding='utf-8')
print('Generated 600 files in', p)

File: scripts/generate_600_changes.js
const fs = require('fs');
const path = require('path');
const dir = path.join(__dirname, '..', 'mass_changes');
if (!fs.existsSync(dir)) fs.mkdirSync(dir, { recursive: true });
for (let i = 1; i <= 600; i++) {
  const name = `change_${String(i).padStart(3, '0')}.txt`;
  const filePath = path.join(dir, name);
  const content = `Change file ${String(i).padStart(3, '0')}\nGenerated for commit #${i}\n`;
  fs.writeFileSync(filePath, content, 'utf8');
}
console.log('Generated 600 files in', dir);
